x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvAppendTests.java

```diff
@@ -188,6 +189,54 @@ private static void bytesRefs(List<TestCaseSupplier> suppliers) {
                 }));
             }
         }
+        suppliers.add(new TestCaseSupplier(List.of(DataType.KEYWORD, DataType.KEYWORD), () -> {
```

Comment:
I think all three test cases here were already added in the `for` loops above. Why are we duplicating them here @jackpan123 ?